kolibri/core/assets/src/kolibri_module.js

```diff
@@ -1,32 +1,33 @@
-'use strict';
 /**
  * KolibriModule module.
  * Provides the base class for Kolibri Modules - all Kolibri Modules must extend this Base class.
  * @module kolibri_module
  */
 
-var Kolibri = require('kolibri');
-var _ = require('lodash');
-var Backbone = require('backbone');
+const Kolibri = require('./core_app');
+const _ = require('lodash');
```

Comment:
We can probably get rid of lodash by using ES6 idioms here.

`_.extend` should be directly replaceable by `Object.assign`.